lib/rex/random_identifier/generator.rb

```diff
@@ -111,11 +112,25 @@ class ExhaustedSpaceError < StandardError; end
     ).freeze
   )
 
+  PHPOpts = DefaultOpts.merge(
+    prefix: '$',
+    first_char_set: Rex::Text::Alpha + '_',
+    # see: https://www.php.net/manual/en/reserved.php
+    # see: https://www.php.net/manual/en/reserved.variables.php
+    forbidden: (
+      %w[
```

Comment:
Some others - that apply to older PHP releases:

```
$HTTP_GET_VARS
$HTTP_POST_VARS
$HTTP_COOKIE_VARS
$HTTP_SERVER_VARS
$HTTP_ENV_VARS
$HTTP_SESSION_VARS
$HTTP_POST_FILES
$HTTP_RAW_POST_DATA
```

They're not on the PHP 7+ docs - but still available on the wayback machine

https://web.archive.org/web/20200210100201/https://www.php.net/manual/en/reserved.variables.httprawpostdata.php